neural_compressor/benchmark.py

```diff
@@ -158,24 +153,21 @@ def __init__(self, conf):
         self._results = {}
         assert isinstance(conf, BenchmarkConfig), \
             "The config object should be config.BenchmarkConfig, not {}".format(type(conf))
-        conf = Config(quantization=None, benchmark=conf, pruning=None, distillation=None, nas=None)
-        self.conf = BenchmarkConf()
-        self.conf.map_pyconfig_to_cfg(conf)
-        if self.conf.usr_cfg.model.framework != 'NA':
-            self.framework = self.conf.usr_cfg.model.framework.lower()
+        self.conf = Config(quantization=None, benchmark=conf, pruning=None, distillation=None, nas=None)
```

Comment:
what's the Config class for? we have had many Config classes, why still need an extra "Config" class? if it's internal class for orchestration, we should use _Config but not Config